sigopt/xgboost/run.py

```diff
@@ -157,7 +157,12 @@ def check_learning_task(self):
 
   def log_feature_importances(self, importance_type='weight', fmap=''):
     scores = self.bst.get_score(importance_type=importance_type, fmap=fmap)
-    scores = dict(sorted(scores.items(), key=lambda x:(x[1], x[0]), reverse=True)[:FEATURE_IMPORTANCES_MAX_NUM_FEATURE])
+    # NOTE: do not log importances if there is no split at all.
+    if not scores:
+      return
```

Comment:
cc @yuhong-sigopt @ericlee0803 , I decided to not log feature importances if none of the tree has a split. Let me know if you are ok with this.

Reply:
Fine w/ me

Reply:
I am ok.